src/main/java/com/manardenza/console/ConsoleMain.java

```diff
@@ -1,16 +1,17 @@
 package com.manardenza.console;
 
+import java.io.IOException;
+
 public class ConsoleMain {
 
-    private static ConsoleMain instance;
+    private ContentsUserMenu contentsUserMenu;
 
-    private ConsoleMain() {
+    public ConsoleMain(ContentsUserMenu contentsUserMenu) {
+        this.contentsUserMenu = contentsUserMenu;
     }
 
-    public static ConsoleMain getInstance() {
-        if (instance == null) {
-            instance = new ConsoleMain();
-        }
-        return instance;
+    public void consoleMain() throws IllegalAccessException, InstantiationException, IOException {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Refactor this method to throw at most one checked exception instead of: java.lang.IllegalAccessException, java.lang.InstantiationException, java.io.IOException [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS1160)